molecule/alternative/prepare.yml

```diff
@@ -35,3 +35,20 @@
         state: link
       run_once: true
       check_mode: false
+
+    - name: get collector scripts
+      become: false
+      get_url:
+        url: "{{ item }}"
+        dest: "/tmp/{{ item.split('/')[-1] }}"
```

Comment:
Let's not deploy scripts to /tmp. If we need a bindir for these, how about `/usr/local/node_exporter/bin/`.

Reply:
This is downloading them for tests only and on deployer machine. Which means there might not be `/usr/local/node_exporter` directory and the access may be denied.

Look at https://github.com/cloudalchemy/ansible-node-exporter/pull/141/files/c4b7f38019f7f8cbb369289f35d5994d9c5b37b6#diff-80a3e3e426b45ed354f680fa3b154ff3R3